demos/connector/neo4j/cluster-gcn-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>It'd be good to refer to the real Cluster-GCN example ( <code>../../node-classification/cluster-gcn-node-classification.ipynb</code> ) in at least one place in this notebook, similar to how the Neo4j + GraphSAGE ones refer to the main GraphSAGE one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1567/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>